packages/expo-cli/src/commands/eject.ts

```diff
@@ -13,7 +13,7 @@ import { learnMore } from './utils/TerminalLink';
 
 async function userWantsToEjectWithoutUpgradingAsync() {
   const answer = await confirmAsync({
-    message: `We recommend upgrading to the latest SDK version before ejecting. SDK 37 introduces support for OTA updates and notifications in ejected projects, and includes many features that make ejecting your project easier. Would you like to continue ejecting anyways?`,
+    message: `We recommend upgrading to the latest SDK version before ejecting. It will be more difficult to upgrade your app after ejecting because you will also be responsible for native iOS and Android related upgrade steps. Continue ejecting?`,
```

Comment:
```suggestion
    message: `We recommend upgrading to the latest SDK version before proceeding. It will be more difficult to upgrade your app after ejecting because you will also be responsible for native iOS and Android related upgrade steps. Continue ejecting?`,
```